rules/windows/registry_event/sysmon_runonce_persistence.yml

```diff
@@ -0,0 +1,25 @@
+title: Run Once task configuration in the registry
+id: c74d7efc-8826-45d9-b8bb-f04fac9e4eff
+description: Rule to detect the configuration of Run Once registry key. Configured payload can be run by runonce.exe /AlternateShellStartup
+author: Avneet Singh '@v3t0_'
```

Comment:
you need to quote the entire `author` field, this way:

```
author: 'Avneet Singh'@v3t0_ oscd.community'
```

Reply:
I've changed it and added the oscd.community too :)